recipes/xerces-c/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+if [ `uname` == Darwin ]; then
+	export CC=clang
+	export CXX=clang++
```

Comment:
Ok, I think yes. Though for C or old C++ it doesn't matter as much. Projects that use C++11 features must use `clang`.